integration/lib/Type.class.php

```diff
@@ -79,7 +79,7 @@ static function createInstance($cl, $args) {
 		if($c === null) {
 			return null;
 		}
-		return $inst = $c->getConstructor() ? $c->newInstanceArgs($args->�a) : $c->newInstanceArgs();
+		return $inst = $c->getConstructor() ? $c->newInstanceArgs($args->»a) : $c->newInstanceArgs();
```

Comment:
Why is this one `Â»a` and not `»a`?

Reply:
@carla-at-wiris do you know if this is relevant?

Reply:
I don't know, but looking at previous commits and other repositories using the same file, this was the `char` used.